pygrocy/data_models/chore.py

```diff
@@ -148,4 +149,4 @@ def track_count(self) -> int:
 
     @property
     def next_execution_assigned_user(self) -> User:
-        return self._next_execution_assigned_user
\ No newline at end of file
+        return self._next_execution_assigned_user
```

Comment:
this looks weird, could you revert this or is it intended?

Reply:
I edited the file directly in github and it looks like it added a newline at the end of the file automatically. A quick google search returned this : https://thoughtbot.com/blog/no-newline-at-end-of-file

To be honest I am not sure how to "undo" this but it looks like having a new line at the end of the file is good practice https://www.flake8rules.com/rules/W292.html